lib/board.pins.js

```diff
@@ -192,40 +190,6 @@ Pins.translations = {
       D4: 4,
       D2: 2
     }
-  },
-  LEONARDO: {
```

Comment:
This was overwritten immediately after to just be a pointer to `Pins.translations.UNO`.